proposals/metrics/prometheus.go

```diff
@@ -48,3 +48,94 @@ var NumBlocksInException = metrics.NewHistogramWithBuckets(
 	},
 	prometheus.ExponentialBuckets(1, 2, 8),
 )
+
+var (
+	// ProposalDuration ...
+	ProposalDuration = metrics.NewHistogramWithBuckets(
```

Comment:
i realized after merging a change to vm that using step as a label is very convenient. it allows to query total hist and breakdown using a single query

Reply:
@dshulyak changed to what you suggested. it's super handy!
![Screenshot from 2022-07-19 16-06-57](https://user-images.githubusercontent.com/30611210/179863011-9865f238-46a0-4e59-a12d-97238496a745.png)

Reply:
also moved the metrics inside proposal package.